starfish/core/types/_constants.py

```diff
@@ -40,6 +41,13 @@ class Coordinates(AugmentedEnum):
 """
 
 
+PerImageSpotResults = namedtuple('PerImageSpotResults', 'spot_attrs, extras')
```

Comment:
wait is this right?  this is just two strings.

Reply:
The strings are just the names of the values put into the named tuple. So you can access them like PerImageSpotResults.spot_attrs

Reply:
Shouldn't it be:

```suggestion
PerImageSpotResults = namedtuple('PerImageSpotResults', 'spot_attrs', 'extras')
```